guides/image-sources/provisioning-installer-bootdisk-fullhost.plantuml

```diff
@@ -0,0 +1,61 @@
+@startuml
+
+!include foreman.pstyle
+
+title Installer-based provisioning with full-host bootdisk
+
+actor       User
+participant "Provisioned\nHost" as Host
+participant "Foreman" as Foreman
+participant "Foreman\nProxy" as Proxy
+participant "Pulp\n(Katello)" as Pulp
+'participant TFTP
+participant DHCP
+participant DNS
+!if ($puppet)
+participant "Puppet\nserver" as Puppet
+!endif
+
+note over Host : powered off
+
+== Create host in Foreman ==
+
+User -> Foreman : clicks **Create Host**
+Foreman -> Proxy : requests free IP address
+Proxy -> DHCP : reads DHCP cofiguration
+note over Foreman : pings IP address
+User -> Foreman : submits the **Create host** form
+group Templates [*PXE* installer boot]
+    Foreman -> Proxy : renders bootloader configuration files
+end
+Proxy -> Proxy : deploys bootloader configuration files
+Foreman -> Proxy : commands to download OS installer kernel\nand initial RAM disk
+Proxy -> Pulp : downloads OS installer kernel\nand initial RAM disk
+Foreman -> Proxy : creates DHCP reservation
+Proxy -> DHCP : forwards DHCP reservation
+Foreman -> Proxy : creates DNS records
+Proxy -> DNS : forwards DNS records
+note over Foreman : host is created\n(build mode enabled)
+
+User -> Foreman : downloads the bootdisk ISO from the host
+note over User : writes the bootdisk ISO\nto a USB/CD/DVD drive
+
+== Boot into OS installer ==
+
+User -> Host : configures the machine to boot\nfrom the USB/CD/DVD drive
+User -> Host : powers on the machine
+Host -> DHCP : requests the reserved IP
+note over Host : boots from the USB/CD/DVD drive
+note over Host : OS installer loads
+User -> Host : eliminates the USB/CD/DVD drive\n(too soon?)
```

Comment:
If we don't know the exact step where it happens, do we have to mention it?

Reply:
Well, they have to do it at some point, otherwise they would be stuck in a loop :D
So we might as well suggest when is the right time.

As it is a necessary step in the process, I would prefer to keep it.